### DISCUSSION TITLE
Can I customize the style of date picker?

### DISCUSSION THREAD

**Discussion by noveliansyah** [henninghall/react-native-date-picker#271] [Q&A] [answered]
is there any way to costumize style?

from this 
![Capture2](https://user-images.githubusercontent.com/63382592/101856433-e3083b00-3b97-11eb-9a02-b6ecc56fe56e.PNG)
You can see that the distance between the month and the date is very close, there is a way to add some padding or margin ?

to this
![Capture](https://user-images.githubusercontent.com/63382592/101856463-ef8c9380-3b97-11eb-9e7c-760d6319fee5.PNG)

**Environment info** 
react-native info output:

`System:
    OS: Windows 10 10.0.19041
    CPU: (4) x64 Intel(R) Core(TM) i3-7020U CPU @ 2.30GHz
    Memory: 1.49 GB / 11.88 GB
  Binaries:
    Node: 10.18.1 - C:\Program Files\nodejs\node.EXE
    npm: 6.13.4 - C:\Program Files\nodejs\npm.CMD
  SDKs:
    Android SDK:
      API Levels: 27, 28, 29
      Build Tools: 28.0.3, 29.0.2, 29.0.3
      System Images: android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: Version  4.1.0.0 AI-201.8743.12.41.6858069
  npmPackages:
    react: 16.13.1 => 16.13.1
    react-native: 0.63.3 => 0.63.3`

 - OS: [Android and iOS]
 - React Native version [0.63.3]
 - react-native-date-picker version [3.2.5]

**Comment by henninghall** [comment, marked as answer]
Unfortunately not, you would have to fork the library and make your own version in that case.
